lib/blocks.php

```diff
@@ -66,8 +66,16 @@ function gutenberg_parse_blocks( $content ) {
 		);
 	}
 
-	$parser = new Gutenberg_PEG_Parser;
-	return $parser->parse( _gutenberg_utf8_split( $content ) );
+	/**
+	 * Filter to allow plugins to replace the server-side block parser
+	 *
+	 * @since 3.7
```

Comment:
Marking so we don't forget to bump this to 3.8.

Reply:
bumped in 66455b461